### ISSUE TITLE
Refuses to render sometimes with valid LaTeX

### ISSUE THREAD

**Issue by dbanet** [susam/mathb#8]
http://mathb.in/39606

**Comment by jcreigh** [comment]
Looks like it's an issue with subscripts after parentheses? Came across the exact same issue. Only works with the first one in a section, any subsequent breaks it.

http://mathb.in/50948

**Comment by jcreigh** [comment]
Seems to be caused by Markdown converting the underscores into <em> tags, since the closing parenthesis is seen as a word boundary.

Not sure of an easy way to fix this.

**Comment by susam** [comment]
The underscores should be escaped with a backslash so that Markdown does not convert them to `<em>` tags.

Example: http://mathb.in/51078
```
$$
(Foo)\_{a} \\
(Foo)\_{a}
$$
```

**Comment by jcreigh** [comment]
But then they're literally underscores instead of being rendered as
subscripts

**Comment by susam** [comment]
I overlooked the fact that the subscripts were not rendered. My bad. Thanks for explaining the issue.

Part of the problem is caused by this line in the JavaScript code (https://github.com/susam/mathb/blob/0433c40cdc5f97dd8e7f9861dcf6b512baa5bcd9/scripts/mathb.js#L229) that tries to preserve the backslashes as literal backslashes. Due to this code, the backslashes in the input no longer function in escape sequences. When I wrote this, I thought it was a good idea. But it seems like a problem now.

By the way, with a quick test on http://math.stackexchange.com/questions/ask, I see that the code at http://mathb.in/51078 that is breaking on MathB.in seems to be rendering fine on Math StackExchange. I'll have to see how they are doing this right.

It will take me a while to get back to this project. So in the meantime, any pull requests are very welcome.
